### ISSUE TITLE
[HELP NEEDED]How to print eval_model's opinion of improving prompt in the Prompt Optimization Example

### ISSUE THREAD

**Issue by IcyFeather233** [zou-group/textgrad#28]
In [Optimizing a Code Snippet and Define a New Loss Example](https://colab.research.google.com/github/zou-group/textgrad/blob/main/examples/notebooks/Tutorial-Test-Time-Loss-for-Code.ipynb), I can use:

```
# Let's do the forward pass for the loss function.
loss = loss_fn(problem, code)
print(loss.value)
```

and

```
# Let's look at the gradients!
loss.backward()
print(code.gradients)
```

to see the prompt's evolution reasons.

But in [Prompt Optimization Example](https://colab.research.google.com/github/zou-group/TextGrad/blob/main/examples/notebooks/Prompt-Optimization.ipynb), I don't know how to modify. I want to see some judgements like "Your step-by-step reasoning is clear and logical, but it contains a critical flaw in the assumption that drying time is directly proportional to the number of shirts. [...]" "your prompt should be improved in ..."

**Comment by vinid** [comment]
Hello! 

You can print `system_prompt.gradients` to see the gradients of the system prompt.

For the the loss you can instead print each of the single `losses` (e.g., `losses[0]`). However, we wrapped the process in various subclasses, so if you want to see the entire thing you should take a look at how we create the [eval_fn](https://github.com/zou-group/textgrad/blob/main/textgrad/tasks/__init__.py) and the [MultiFieldTokenParsedEvaluation](https://github.com/zou-group/textgrad/blob/main/textgrad/loss.py#L114).
